### ISSUE TITLE
General error "TypeError"

### ISSUE THREAD

**Issue by x829901** [avantgardefinance/melon-terminal#586]
TypeError: e.comparedTo is not a function
    at https://melon.avantgarde.finance/static/js/1.9a5d31e6.chunk.js:1:19526
    at as (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2722585)
    at yc (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2741726)
    at t.unstable_runWithPriority (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2760045)
    at Ha (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2684172)
    at bc (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2741483)
    at Xs (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2733001)
    at https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2684463
    at t.unstable_runWithPriority (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2760045)
    at Ha (https://melon.avantgarde.finance/static/js/8.3f440d44.chunk.js:2:2684172)

**Comment by fubhy** [comment]
Hi. Can you provide some additional details? Which page did this error occur on? What is the fund address? Thanks!

**Comment by x829901** [comment]
It happened from within the fund manager trade page. There are three options currently: Kyber, Uniswap, and 0x and each option provides only partial information. To see the actual cost of the trade, I have to click each option to buy to see what the fees would be if I used Kyber, Uniswap, or 0x. I then subtract the fee from how much coin I'd be receiving after the trade and I pick the liquidity pool that nets me the greatest number. This requires me to manually click each option to buy. I first click on the "Buy" button for Kyber Network, I record the fee in a spreadsheet, then I close the dialog box. Then I click on Uniswap option to buy, record the fee in a spreadsheet, and upon closing this dialog box, each and pretty much every time, the dialog box crashes with the error posted here, with a link for the error to be posted as an issue. So, I did just that. Sorry if I didn't post with some context. I'm sure creating a feature that would show the received token amount MINUS transaction fees would allow fund managers to see the _actual_ cost to trade. The act of clicking each option to buy from the different pools is what throws up the error.
